### PR TITLE
add refresh design sub nav styles

### PR DESCRIPTION
**Pull request by 5h1rU**
The sub-navigation bar changes are entirely CSS changes. 

This look and feel are notably similar to the tabs component. The main difference is how the a11y behaves. Sub nav is only navigable with the tab key as any navigation list of links.

This time the idea was to modify the original bootstrap nav-tabs items from the `nav.css` file. It's being handled in global styles meaning the change will be general in any navigation tab section across the codebase.

Sub nav with icons isn't modified since this is not part of the tabs per se. Any change around icons in the sub-nav is feasible, and the only step needed is to remove or add icons as you need.

Below is the video with a small demo on how this change is applied.

https://user-images.githubusercontent.com/989826/118712466-6d5b6780-b7e6-11eb-8530-43778adb1167.mov

### CODE REVIEW

**Review by limitedmage — APPROVED** [review]

File: client/branded/src/global-styles/nav.scss [hunk 1]
```diff
@@ -13,6 +13,24 @@ $nav-tabs-link-active-border-color: var(--border-color) var(--border-color) tran
...
+            margin-top: 0;
+            color: var(--body-color);
+        }
+    }
+}
+
+.nav-tabs > .nav-item > .nav-link.active {
+    .theme-redesign & {
+        color: var(--body-color);
+        font-weight: bold;
```
Comment (valerybugakov) [inline client/branded/src/global-styles/nav.scss:31]: It seems that according to [mockups](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1107%3A23176), it should be `600`.

```suggestion
        font-weight: 600;
```

File: client/branded/src/global-styles/nav.scss [hunk 2]
```diff
@@ -13,6 +13,24 @@ $nav-tabs-link-active-border-color: var(--border-color) var(--border-color) tran
 // to have an undesirable bottom border when active.
 .nav-tabs > .nav-item > .nav-link {
     height: 100%;
+    .theme-redesign & {
+        border: none;
+        border-radius: var(--border-radius);
+        color: var(--text-muted);
+        &:not(.active):hover {
+            border-bottom: 3px solid var(--border-color-2);
```
Comment (valerybugakov) [inline client/branded/src/global-styles/nav.scss:21]: Based on [mockups](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1107%3A23176):
```suggestion
            border-bottom: 2px solid var(--border-color-2);
```

File: client/branded/src/global-styles/nav.scss [hunk 3]
```diff
@@ -13,6 +13,24 @@ $nav-tabs-link-active-border-color: var(--border-color) var(--border-color) tran
 // to have an undesirable bottom border when active.
 .nav-tabs > .nav-item > .nav-link {
     height: 100%;
+    .theme-redesign & {
+        border: none;
+        border-radius: var(--border-radius);
```
Comment (valerybugakov) [inline client/branded/src/global-styles/nav.scss:18]: Should we apply border radius only to the outline border using `:focus-visible`?

Implementation:

Screenshot 2021-05-19 at 09 39 14

Design:

Screenshot 2021-05-19 at 09 40 39

File: client/branded/src/global-styles/nav.scss [hunk 4]
```diff
@@ -13,6 +13,24 @@ $nav-tabs-link-active-border-color: var(--border-color) var(--border-color) tran
 // to have an undesirable bottom border when active.
 .nav-tabs > .nav-item > .nav-link {
     height: 100%;
+    .theme-redesign & {
```
Comment (valerybugakov) [inline client/branded/src/global-styles/nav.scss:16]: Probably not introduced in this PR: a general issue with outline border when second tab is active: 

Screenshot 2021-05-19 at 09 38 59

**Review by valerybugakov — APPROVED** [review]
Noticed a couple of inconsistencies with mockups. 

I might be looking into the wrong Figma frame because the PR doesn't have a link. If that's the case feel free to go ahead with the merge. 🙂

**Comment by felixfbecker** [comment]
> This look and feel are notably similar to the tabs component. The main difference is how the a11y behaves. Sub nav is only navigable with the tab key as any navigation list of links.

I was curious what the reasoning behind this was/if this is correct (it is). Sharing the quote here for others:

> Note that navigation bars, even if visually styled as tabs with the .nav-tabs class, should not be given role="tablist", role="tab" or role="tabpanel" attributes. These are only appropriate for dynamic tabbed interfaces, as described in the WAI ARIA Authoring Practices.

From [Bootstrap docs](https://getbootstrap.com/docs/5.0/components/navs-tabs/#:~:text=note%20that%20navigation%20bars%2C%20even%20if%20visually%20styled%20as%20tabs%20with%20the%20.nav-tabs%20class%2C%20should%20not%20be%20given%20role%3D%22tablist%22%2C%20role%3D%22tab%22%20or%20role%3D%22tabpanel%22%20attributes.%20these%20are%20only%20appropriate%20for%20dynamic%20tabbed%20interfaces%2C%20as%20described%20in%20the%20wai%20aria%20authoring%20practices.) (which links to the ARIA spec).

**Pull request merged**